config/initializers/content_security_policy.rb

```diff
@@ -1,26 +1,91 @@
 # Be sure to restart your server when you modify this file.
-
+#
 # Define an application-wide content security policy
 # For further information see the following documentation
-# https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+#
+# - {https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy}
+# - {https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP for more CSP info}
+#
+# The resulting policy should be checked with:
+#
+# - {https://csp-evaluator.withgoogle.com}
+# - {https://cspvalidator.org}
 
-Rails.application.configure do
-  config.content_security_policy do |policy|
-    policy.default_src :self, :https
-    policy.font_src    :self, :https, :data
-    policy.img_src     :self, :https, :data
-    policy.object_src  :none
-    policy.script_src  :self, :https
-    policy.style_src   :self, :https
-    # Specify URI for violation reports
-    # policy.report_uri "/csp-violation-report-endpoint"
-  end
+GOVUK_DOMAINS = [
+  '*.education.gov.uk',
+  '*.s3.eu-west-2.amazonaws.com',
+  'eyfs-cms-preprod.london.cloudapps.digital',
```

Comment:
This is reform related

Reply:
Yes however for now content team want to host images on reform and use the link here

Reply:
Removed reform links